.github/workflows/link-check-all.yml

```diff
@@ -10,9 +10,7 @@ jobs:
     env:
       NODE_OPTIONS: '--max-http-header-size=65536'
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Run Link Check
-        uses: 'iterative/link-check.action@v0.11'
-        with:
-          configFile: 'config/link-check/config.yml'
-          output: consoleLog
+        id: check
+        run: npx repo-link-check -d -c config/link-check/config.yml
```

Comment:
Should we use echo here for showing output to the console or will it just work?

Reply:
I believe it will just work the same as before, we use `yarn run` without any sort of echo or piping and see their console output.